### PR TITLE
rename fee variables

### PR DESCRIPTION
**Pull request by wakamex**
ran the following commands in OSX:
```
LC_ALL=C find ./ -type f -name "*.py" -exec sed -i '.bak' -e 's/trade_fee_percent/curve_fee_mult/g' {} \;
LC_ALL=C find ./ -type f -name "*.py" -exec sed -i '.bak' -e 's/redemption/flat/g' {} \;
LC_ALL=C find ./ -type f -name "*.py" -exec sed -i '.bak' -e 's/governance_fee_percent/governance_fee_mult/g' {} \;
LC_ALL=C find ./ -type f -name "*.py" -exec sed -i '.bak' -e 's/redemption/flat/g' {} \;
LC_ALL=C find ./ -type f -name "*.ipynb" -exec sed -i '.bak' -e 's/trade_fee_percent/curve_fee_mult/g' {} \;
LC_ALL=C find ./ -type f -name "*.ipynb" -exec sed -i '.bak' -e 's/redemption_fee_percent/flat_fee_mult/g' {} \;
LC_ALL=C find ./ -type f -name "*.ipynb" -exec sed -i '.bak' -e 's/governance_fee_percent/governance_fee_mult/g' {} \;
LC_ALL=C find ./ -type f -name "*.ipynb" -exec sed -i '.bak' -e 's/redemption_fee/flat_fee/g' {} \;
```

then after
```
LC_ALL=C find ./ -type f -name "*.py" -exec sed -i '.bak' -e 's/flat_fee_percent/flat_fee_mult/g' {} \;
LC_ALL=C find ./ -type f -name "*.ipynb" -exec sed -i '.bak' -e 's/flat_fee_percent/flat_fee_mult/g' {} \;
```

### CODE REVIEW

File: tests/pricing_models/test_calc_in_given_out.py [hunk 1]
```diff
@@ -131,9 +131,9 @@ def test_calc_in_given_out_precision(self):
                     bond_reserves=10_000_000_000,
                     share_price=2,
                     init_share_price=1.2,
-                    trade_fee_percent=0.1,
+                    curve_fee_mult=0.1,
                     # TODO: test with redemption fees
```
Comment (dpaiton) [inline tests/pricing_models/test_calc_in_given_out.py:135]: redemption

File: tests/pricing_models/test_calc_in_given_out.py [hunk 2]
```diff
@@ -218,8 +218,8 @@ def test_calc_in_given_out_failure(self):
                     share_price=1,
                     init_share_price=1,
                     # trade fee negative
```
Comment (dpaiton) [inline tests/pricing_models/test_calc_in_given_out.py:220]: trade

File: tests/pricing_models/test_calc_in_given_out.py [hunk 3]
```diff
@@ -231,9 +231,9 @@ def test_calc_in_given_out_failure(self):
                     bond_reserves=1_000_000,
                     share_price=1,
                     init_share_price=1,
-                    trade_fee_percent=0.01,
+                    curve_fee_mult=0.01,
                     # redemption fee negative
```
Comment (dpaiton) [inline tests/pricing_models/test_calc_in_given_out.py:235]: redemption

File: tests/pricing_models/test_calc_in_given_out.py [hunk 4]
```diff
@@ -246,8 +246,8 @@ def test_calc_in_given_out_failure(self):
                     share_price=1,
                     init_share_price=1,
                     # trade fee above 1
```
Comment (dpaiton) [inline tests/pricing_models/test_calc_in_given_out.py:248]: trade

Comment (dpaiton) [reply tests/pricing_models/test_calc_in_given_out.py:248]: I'll not comment on the rest -- there are more in this file.

Comment (dpaiton) [reply tests/pricing_models/test_calc_in_given_out.py:248]: also "fee percent" comments

**Review by dpaiton — CHANGES_REQUESTED** [review]
I'm wondering what the plan is long-term: should we have a set of "fee" variables per market? will they all be specified as individual parameters? or maybe a "fees" data class or dictionary?

**Review by wakamex — COMMENTED** [review]
lgtm, read through all the diffs

**Pull request merged**